config/webpack/webpack.config.ssr.js

```diff
@@ -295,15 +297,17 @@ const buildWebpackConfigs = builds.map(
             }
           ]
         },
-        plugins: isStartScript
-          ? [
-              new StartServerPlugin('server.js'),
-              new webpack.NamedModulesPlugin(),
-              new webpack.HotModuleReplacementPlugin(),
-              new webpack.NoEmitOnErrorsPlugin(),
-              new webpack.DefinePlugin(envVars)
-            ]
-          : [new webpack.DefinePlugin(envVars)]
+        plugins: [new webpack.DefinePlugin(envVars)].concat(
```

Comment:
Wouldn't this mean DefinePlugin is added twice if `isStartScript` is truthy? Is that intentional?

Reply:
I am sorry. I forgot to delete the first one. Deleting it